packages/sources/coinranking/README.md

```diff
@@ -6,65 +6,85 @@ The adapter takes the following environment variables:
 
 | Required? |   Name    | Description | Options | Defaults to |
 | :-------: | :-------: | :---------: | :-----: | :---------: |
-|    ✅     | `API_KEY` |             |         |             |
+|           | `API_KEY` |             |         |             |
 
-## Input Params
+### Input Parameters
 
-- `base`, `from`, or `coin`: The symbol of the currency to query
-- `quote`, `to`, or `market`: The symbol of the currency to convert to
+| Required? |   Name   |     Description     |                          Options                           | Defaults to |
+| :-------: | :------: | :-----------------: | :--------------------------------------------------------: | :---------: |
+|           | endpoint | The endpoint to use | [price](#Price-Endpoint), [marketcap](#Marketcap-Endpoint) |   `price`   |
 
-## Output
+---
+
+## Price Endpoint
+
+https://api.coinranking.com/v2/coin/`{COIN_UUID}`
+
+### Input Params
+
+| Required? |          Name          |                   Description                    | Options | Defaults to |
+| :-------: | :--------------------: | :----------------------------------------------: | :-----: | :---------: |
+|    ✅     | `base`, `from`, `coin` |       The symbol of the currency to query        |         |             |
+|    🟡     |        `coinid`        | The coin ID (optional to use in place of `base`) |         |             |
+
+### Sample Input
+
+```json
+{
+  "id": "1",
+  "data": {
+    "base": "ETH"
+  }
+}
+```
+
+### Sample Output
+
+```json
+{
+  "jobRunID": "1",
+  "data": {
+    "result": 2110.2357344391503
+  },
+  "result": 2110.2357344391503,
+  "statusCode": 200
+}
+```
+
+---
+
+## Marketcap Endpoint
+
+https://api.coinranking.com/v2/coin/`{COIN_UUID}`
+
+### Input Params
+
+| Required? |          Name          |                   Description                    | Options | Defaults to |
+| :-------: | :--------------------: | :----------------------------------------------: | :-----: | :---------: |
+|    ✅     | `base`, `from`, `coin` |       The symbol of the currency to query        |         |             |
+|    🟡     |        `coinid`        | The coin ID (optional to use in place of `base`) |         |             |
+
+### Sample Input
+
+```json
+{
+  "id": "1",
+  "data": {
+    "endpoint": "marketcap",
+    "base": "ETH"
+  }
+}
+```
+
+### Sample Output
 
 ```json
 {
   "jobRunID": "1",
   "data": {
-    "uuid": "razxDUgYGNAdQ",
```

Comment:
Could we get this code-ified into an interface in `coin.ts` endpoint?

I originally started putting the full verbose response here for easy reference when changing the adapter, but agree that it makes the README very lengthy.